coderedcms/templates/coderedcms/pages/article_page.html

```diff
@@ -30,6 +30,11 @@
 {% if settings.coderedcms.SeoSettings.twitter_meta %}
     {% block twitter_card %}{% if self.cover_image %}summary_large_image{% else %}{{block.super}}{% endif %}{% endblock %}
     {% block twitter_seo_extra %}
+        <meta name="twitter:title" content="{{ self.title }}">
+        {% if self.caption %}
+        <meta name="twitter:description" content="{{ self.caption }}">
+        {% endif %}
+        <meta name="twitter:image" content="{{self.get_site.root_url}}{{ twitter_card_image self }}">
```

Comment:
I don't think we need to provide the `twitter:title` or `twitter:image` here, they are already provided in `base.html`